packages/lexical-utils/src/positionNodeOnRange.ts

```diff
@@ -20,7 +20,27 @@ const mutationObserverConfig = {
   subtree: true,
 };
 
-export default function positionNodeOnRange(
+function prependDOMNode(parent: HTMLElement, node: HTMLElement) {
+  if (parent.hasChildNodes()) {
+    parent.insertBefore(node, parent.firstChild);
+  } else {
+    parent.append(node);
+  }
```

Comment:
null already means append, so you don't need to check it https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore#referencenode

```suggestion
  parent.insertBefore(node, parent.firstChild);
```